src/widgets/stats/stats.js

```diff
@@ -42,6 +44,11 @@ const renderer = ({ containerNode, cssClasses, renderState, templates }) => (
       cssClasses={cssClasses}
       hitsPerPage={hitsPerPage}
       nbHits={nbHits}
+      nbSortedHits={nbSortedHits}
+      isSmartSorted={
+        typeof appliedRelevancyStrictness !== 'undefined' &&
+        appliedRelevancyStrictness > 0
+      }
```

Comment:
this condition should be in the connector, and probably be more like `typeof helper.state.relevancyStrictness !== 'undefined'`, wdyt?

Reply:
If it's `0`, it's not smartly sorted. There are two cases when we get `0` as `relevancyStrictness`

1. customers set it like that on dashboard (meaning they don't want smart sort on the index)
2. a user clicked "see all results" button from <SmartSort /> and InstantSearch attached `relevancyStrictness=0` to the search parameter to suppress the smart setting (even it was configured on dashboard).

Does it make sense?

Reply:
I see, that means that you should probably remove the option to set a value for relevancyStrictness in the widget, so it always toggles between 0 and undefined, wdyt? 

still think this condition should be in the connector

Reply:
https://github.com/algolia/instantsearch.js/pull/4649/commits/bf2d1694d19f9eb4d928dbd9cbb83cb201de3200